scripts/RetentionScript/README.md

```diff
@@ -0,0 +1,68 @@
+
+# About the scripts:
+These scripts are used to cleanup the storage configured for BRS service in service fabric. It allows you to delete data older than specific date time.
+
+## Usage
+This script could be used in case of both type of supported storages:
+1) Azure blob store
+2) File Share
+ How to use with Azure Storage:
+ You can run the script by providing single azure storage connection string as parameter or you can also provide azure account name and key separately.
+ Examples:
+1) The below example will delete all the backups in the storage(specifed with connection string) of the service with serviceid "WebReferenceApplication~RestockRequestManager".
+```powershell
+ .\RetentionScript.ps1 -DateTimeBefore "2018-06-18 23.44.03Z" -ConnectionString "your-connection-string" -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -SSLCertificateThumbPrint "Client#Certificate#Thumbpring" -ServiceId "WebReferenceApplication~RestockRequestManager"
+```
+2) In this example, passing storage account name and key separately. 
+```powershell
+ .\RetentionScript.ps1 -DateTimeBefore "2018-06-18 23.44.03Z" -StorageAccountName "storageaccountname" -StorageAccountName "storgeAccountName" -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -SSLCertificateThumbPrint "Client#Certificate#Thumbpring" -ServiceId "WebReferenceApplication~RestockRequestManager"
+ ```
+ 3) You can also filter the container for cleanup. This example will delete backup data for all the partitions available on the container with container name "brstorage":
+```powershell
+ .\RetentionScript.ps1 -ContainerName "brstorage"-DateTimeBefore "2018-06-18 23.44.03Z" -StorageAccountName "storageaccountname" -StorageAccountName "storgeAccountName" -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -SSLCertificateThumbPrint "Client#Certificate#Thumbpring"
+ ```
+ 3) Similarly you can also filter the application by providing the ApplicationId. If Application name is "WebReferenceApplication/RestockRequestManager"(without fabric:) then,
+ 	  application id is "WebReferenceApplication~RestockRequestManager":
```

Comment:
I am confused now. I thought you were using ~ to separate application and service name